### PR TITLE
Rotmat :fix python3 divisions

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

**Comment by khancyr** [comment]
@peterbarker ping ! Simple fix and covered by unittest !

File: tests/test_rotmat.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def test_maths(self):
         assert v1 - v2 == Vector3(0, -1, -6)
         assert (v1 * 3) == Vector3(3, 6, -9)
         assert v1 * v2 == -2
+        assert v2 / 2.0 == Vector3(0.5, 1.5, 1.5)
```
Comment (cclauss) [inline tests/test_rotmat.py:47]: Also, add tests for true_div?

File: tests/test_rotmat.py [hunk 2]
```diff
@@ -77,6 +78,7 @@ def test_maths(self):
         assert m1 - m2 == Matrix3(Vector3(0, 0, 0), Vector3(1, 4, 0), Vector3(1, 0, -8))
         assert m1 * 3 == Matrix3(Vector3(3, 0, 0), Vector3(3, 15, 0), Vector3(3, 0, -21))
         assert m1 * m1 == Matrix3(Vector3(1, 0, 0), Vector3(6, 25, 0), Vector3(-6, 0, 49))
+        assert m1 / 2.0 == Matrix3(Vector3(0.5, 0, 0), Vector3(0.5, 2.5, 0), Vector3(0.5, 0, -3.5))
```
Comment (cclauss) [inline tests/test_rotmat.py:81]: Also, add tests for true_div?

File: tests/test_rotmat.py [hunk 3]
```diff
@@ -109,7 +111,7 @@ def test_matrixops(self):
         m1 = Matrix3(Vector3(1, 0, 0), Vector3(1, 5, 0), Vector3(1, 0, -7))
 
         m1.normalize()
-        print(m1)
+        #print(m1)
```
Comment (cclauss) [inline tests/test_rotmat.py:114]:
```suggestion
        # print(m1)
```

File: tests/test_rotmat.py (hunk 1 above)
Comment (khancyr) [reply tests/test_rotmat.py:47]: this is already true_div, you mean floor_div  ?

**Comment by peterbarker** [comment]
Squashed it up and fixed the commit message.

File: tests/test_rotmat.py [hunk 4]
```diff
@@ -44,6 +44,8 @@ def test_maths(self):
         assert v1 - v2 == Vector3(0, -1, -6)
         assert (v1 * 3) == Vector3(3, 6, -9)
         assert v1 * v2 == -2
+        assert v2 / 2.0 == Vector3(0.5, 1.5, 1.5)
+        assert v2 // 2.0 == Vector3(0, 1, 1)
```
Comment (cclauss) [inline tests/test_rotmat.py:48]:
```suggestion
        assert v2 / 2.0 == Vector3(0.5, 1.5, 1.5)
        assert v2 // 2.0 == Vector3(0, 1, 1)
        assert v2 / 2.1 == Vector3(0.23809523809523808, 0.7142857142857143, 0.7142857142857143)
        assert v2 // 2.1 == Vector3(0.0, 0.0, 0.0)
```
Please add tests with non-whole number divisors.

File: tests/test_rotmat.py [hunk 5]
```diff
@@ -77,6 +79,7 @@ def test_maths(self):
         assert m1 - m2 == Matrix3(Vector3(0, 0, 0), Vector3(1, 4, 0), Vector3(1, 0, -8))
         assert m1 * 3 == Matrix3(Vector3(3, 0, 0), Vector3(3, 15, 0), Vector3(3, 0, -21))
         assert m1 * m1 == Matrix3(Vector3(1, 0, 0), Vector3(6, 25, 0), Vector3(-6, 0, 49))
+        assert m1 / 2.0 == Matrix3(Vector3(0.5, 0, 0), Vector3(0.5, 2.5, 0), Vector3(0.5, 0, -3.5))
```
Comment (cclauss) [inline tests/test_rotmat.py:82]: Please add tests with non-whole number divisors.

**Review by cclauss — CHANGES_REQUESTED** [review]

**Comment by peterbarker** [comment]
@khancyr @cclauss has asked for a couple more test cases be added.  Shouldn't take too long.

**Comment by khancyr** [comment]
@peterbarker done

**Comment by cclauss** [comment]
Travis jobs seem to be taking an eternity to run.  Perhaps it is time to migrate off of https://travis-ci.org and onto https://travis-ci.com.  (Or GitHub Actions or CircleCI, etc.)

**Review by peterbarker — APPROVED** [review]

**Comment by peterbarker** [comment]
Merged, thanks @khancyr , @cclauss

**Comment by peterbarker** [comment]
@cclauss So my reading seems to indicate travis-ci.org is essentially legacy and we really ought to shoft over to travis-ci.com as there's no downsides and might be some upsides?

**Comment by peterbarker** [comment]
Ah, to answer my own question - no, it's still in beta: `Your account needs to be part of the beta to start migrating public repositories here. Please, sign up for the beta below`

**Comment by cclauss** [comment]
It is a long-running beta...  Two years and still running.

**Pull request merged**